Libplanet/Net/Swarm.cs

```diff
@@ -1381,6 +1414,19 @@ CancellationToken cancellationToken
                 if (pong.AppProtocolVersion != _appProtocolVersion)
```

Comment:
Do we still need the `AppProtocolVersion` in the `Pong`?

Reply:
I think the `Pong` should have `AppProtocolVersion` so that the sender can know the version of the `Peer`.